src/momento/internal/aio/_add_header_client_interceptor.py

```diff
@@ -16,9 +16,8 @@ def __init__(self, name: str, value: str):
 
 
 class AddHeaderStreamingClientInterceptor(grpc.aio.UnaryStreamClientInterceptor):
-    are_only_once_headers_sent = False
-
```

Comment:
Good find. These were class variables like this. If this were a data class, they would be instance variables.